tests/e2e/vmgroups_test.go

```diff
@@ -0,0 +1,307 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test is going to create volume on the fresh testbed very first time.
+// After installing vmdk volume plugin/driver, volume creation should not be
+// failed very first time.
+
+// This test is going to cover the issue reported at #656
+// TODO: as of now we are running the test against photon vm it should be run
+// against various/applicable linux distros.
+
+package e2e
+
+import (
+	"os"
+	"time"
+	. "gopkg.in/check.v1"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+	"github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+)
+
+const (
+	volName1                 = "_test_vol_1"
+	volName2                 = "_test_vol_2"
+	defaultVg                = "_DEFAULT"
```

Comment:
with https://github.com/vmware/docker-volume-vsphere/pull/1283/files#diff-a3a1723e96c4aa2a131a49f75ebd95e0R45 .. we will be getting as admincli constant, please avoid using local test constants.